examples/verify_signature/src/main.sw

```diff
@@ -0,0 +1,66 @@
+contract;
+
+use std::hash::*;
+use std::ecr::{ec_recover_address, EcRecoverError};
+use std::bytes::Bytes;
+use std::b512::B512;
+use std::constants::ZERO_B256;
+
+abi VerifySignature {
+    fn get_message_hash(_to: b256, _amount: u64, _message: str, _nonce: u64) -> b256;
+    fn get_eth_signed_message_hash(_message_hash: b256) -> b256;
+    fn recover_signer(_eth_signed_message_hash: b256, _signature: B512) -> b256;
+    // NEEDS TO BE FIXED
```

Comment:
What needs to be fixed?

Reply:
There was a bug when I first pushed it. Will remove the comment